build_aarch64_wheel.py

```diff
@@ -214,15 +215,19 @@ def install_condaforge_python(host: RemoteHost, python_version="3.8") -> None:
     else:
         install_condaforge(host)
         # Pytorch-1.10 or older are not compatible with setuptools=59.6 or newer
-        host.run_cmd(f"conda install -y python={python_version} numpy pyyaml setuptools=59.5.0")
-
+        host.run_cmd(f"conda install -y python={python_version} numpy pyyaml setuptools=59.5.0; python3 --version")
```

Comment:
Why this is necessary. And please keep 2 lines separator between functions/classes but one line between class methods